parsing.py

```diff
@@ -0,0 +1,32 @@
+import av
+import os
+
```

Comment:
We don't need this file, this is outdated from video encoding methods tested before.